pkg/report/profit_report.go

```diff
@@ -12,134 +11,104 @@ import (
 
 // AccumulatedProfitReport For accumulated profit report output
 type AccumulatedProfitReport struct {
-	// AccumulatedProfitMAWindow Accumulated profit SMA window, in number of trades
-	AccumulatedProfitMAWindow int `json:"accumulatedProfitMAWindow"`
+	// ProfitMAWindow Accumulated profit SMA window
+	ProfitMAWindow int `json:"profitMAWindow"`
 
-	// IntervalWindow interval window, in days
-	IntervalWindow int `json:"intervalWindow"`
-
-	// NumberOfInterval How many intervals to output to TSV
-	NumberOfInterval int `json:"NumberOfInterval"`
+	// ShortTermProfitWindow The window to sum up the short-term profit
+	ShortTermProfitWindow int `json:"shortTermProfitWindow"`
 
 	// TsvReportPath The path to output report to
 	TsvReportPath string `json:"tsvReportPath"`
 
-	// AccumulatedDailyProfitWindow The window to sum up the daily profit, in days
-	AccumulatedDailyProfitWindow int `json:"accumulatedDailyProfitWindow"`
+	symbol string
```

Comment:
why apply private field here?

Reply:
The symbol of the profit report is determined by the strategy